### DISCUSSION TITLE
Issues with macOS 10.15.7 VS 11.7.1 or 12.5

### DISCUSSION THREAD

**Discussion by jacksonhcb** [acids-ircam/RAVE#144] [General]
Hello,
Failed to open standalone on macOS 10.15.7 (message type « L’application requiert macOS 11.6 ou ultérieur » - excuse my French) but also failed to open on macOS 11.7.1 (« Vérifiez auprès du développeur que Rave fonctionne avec cette version de macOS » - on a 2015 MacBook Pro)
Was also unable to compile on a M1 2020 MacBook Air.
If anyone is confident in how to install and use on both these machines, I would happily remunerate your help !!

**Comment by hakanjonsson** [comment]
Hi, I also have problems compiling for MacOS on M1.

I am trying to compile RAVE VST for MacOS on M1 and am running into what I think is namespace problems:
/rave_vst-main/build/torch/libtorch/include/ATen/DeviceGuard.h:26:49: ::error: error: 49reference to ‘nullopt’ is ambiguous reference to ‘nullopt’ is ambiguous error: :
return t.has_value() ? device_of(t.value()) : nullopt; return t.has_value() ? device_of(t.value()) : nullopt;

This seems to be caused by namespace problems between std and juce. I am using cmake 3.25.1, MacOS 13.0.1 and clang-14.0.0. Any suggestions on how to resolve this? Or can anyone who compiled successfully share the binaries?
Thanks,
Håkan
